plugins/ocp/ocp-nvme.c

```diff
@@ -828,6 +828,434 @@ static int eol_plp_failure_mode(int argc, char **argv, struct command *cmd,
 	return err;
 }
 
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Maybe trim the block length to 80 chars. Still pondering if we should split
the file if this is needed. IIRC I was the one who forced you to move all bits
into one file... So I can't really complain about this then, ay? ;)